src/style.css

```diff
@@ -1 +1,86 @@
-/* add styles here */
+@font-face {
+  font-family: Avenir, Arial, sans-serif;
```

Comment:
The `font-family` property inside a `@font-face` rule doesn't accept a list of fonts. It should contain only the name of the font that's being loaded with the rule. The fallback fonts should be listed in the `font-family` property in your CSS rules, not in the `@font-face` rule. For example, you can write it like this: `font-family: 'Avenir', Arial, sans-serif;` in your CSS rules.